docs/2.4/admin-guide.md

```diff
@@ -0,0 +1,1479 @@
+    The commercial Teleport edition called "Teleport Enterprise" supports native
+    audit log exporting into external systems like Splunk, AlertLogic and others.
+    Take a look at [Teleport Enterprise](enterprise.md) section to learn more.
+
+### Recorded Sessions
+
+In addition to logging `session.start` and `session.end` events, Teleport also records the entire
+stream of bytes going to/from standard input and standard output of an SSH session.
+
+The recorded sessions are stored as raw bytes in the `sessions` directory under `log`.
+Each session consists of two files, both are named after the session ID:
+
+1. `.bytes` file represents the raw session bytes and is somewhat
+    human-readable, although you are better off using `tsh play` or
+    the Web UI to replay it.
+2. `.log` file contains the copies of the event log entries that are related
+   to this session.
+
+```bash
+$ ls /var/lib/teleport/log/sessions/default
+-rw-r----- 1 root root 506192 Feb 4 00:46 4c146ec8-eab6-11e6-b1b3-40167e68e931.session.bytes
+-rw-r----- 1 root root  44943 Feb 4 00:46 4c146ec8-eab6-11e6-b1b3-40167e68e931.session.log
+```
+
+To replay this session via CLI:
+
+```bash
+$ tsh --proxy=proxy play 4c146ec8-eab6-11e6-b1b3-40167e68e931
+```
+
+### Recording Proxy Mode
+
+See [Audit Log Architecture](architecture/#audit-log) to understand how the session
+recording works. The important detail is that by default the recording is not
+available if a cluster runs `sshd`, the OpenSSH server, on the nodes.
+
+To enable session recording for `sshd` nodes, the cluster must be switched to
+"recording proxy" mode, in this mode the recording will be done on the proxy level:
+
+```bash
+# snippet from /etc/teleport.yaml
+auth_service:
+   session_recording: "proxy"  # can also be "off" and "on" (default)
+```
+
+Next, `sshd` must be told to allow users to log in with certificates generated by the
+Teleport User CA. Start by exporting the Teleport CA public key:
+
+```bash
+$ tctl auth export --type=user
+```
+
+To allow access to a single user, copy the above output to `~/.ssh/authorized_keys`, but
+to apply this for all users, remove "cert-authority" from the start of `tctl` output and
+copy it to `/etc/ssh/teleport_user_ca.pub`. 
+
+Add the following line to `/etc/ssh/sshd_config`: 
+
+```bash
+TrustedUserCAKeys /etc/ssh/teleport_user_ca.pub
+```
+
+Now `sshd` will trust users who present a Teleport-issued certificate. Next step 
+is to configure host authentication.
+
+When in recording mode, Teleport will check that the host certificate of the
+node a user connects to is signed by a Teleport CA. By default this is a strict
+check, if the node presents just a key, or a certificate signed by a different
+CA, Teleport will reject this connection with the error message saying 
+_"ssh: handshake failed: remote host presented a public key, expected a host certificate"_
+
+You can disable strict host checks as shown below. Note: this opens the possibility for 
```

Comment:
We should include reasons for why you might want to enable this. For example if you are migrating a cluster to Teleport, you may still want to allow access so you can parse the logs to find out which nodes are still presenting keys instead of certificates.